packages/office-ui-fabric-react/src/components/List/List.Props.ts

```diff
@@ -3,7 +3,17 @@ import { IRectangle } from '../../Utilities';
 import { List } from './List';
 
 export interface IList {
-
+  /**
+   * Scroll to the given index. By default will bring the page the specified item is on into the view. If a callback
+   * to measure the height of an individual item is specified, will only scroll to bring the specific item into view.
```

Comment:
Ok, so this function was usable in the past, just wasn't specified in an interface.